styles/sass/sf-additional.scss

```diff
@@ -2361,4 +2361,14 @@ sd-multi-image-edit {
         justify-content: flex-end;
         gap: var(--gap--small);
     }
+}
+
+.modal.modal--tabs .modal__body-footer .button-group{
```

Comment:
I don't think you need all of this, most of it already comes from the framework. Only 'display: flex' is needed.
`.modal.modal--tabs .modal__body-footer .button-group {
    display: flex;
}`
If you add the helper class described in the previous comment you don't need the 'span' properties as well.